lib/matplotlib/axes/_base.py

```diff
@@ -4286,6 +4289,13 @@ def get_tightbbox(self, renderer, call_axes_locator=True,
         -------
         bbox : `.BboxBase`
             bounding box in figure pixel coordinates.
+
+        See Also
+        --------
+        matplotlib.axis.Axes.get_window_extent
```

Comment:
These  extra 'See also' sections with non functional API links are breaking more strict docs builds downstream.
https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.axes.Axes.get_tightbbox.html#matplotlib.axes.Axes.get_tightbbox

(We run the docs build in our CI in astropy with the `-w` option, so any sphinx warning is causing a failure (as our WCSAxes class is inherited from `Axes` and ultimately from this docstring).

If it's helpful I can open a separate issue for this.

Reply:
A new issue would get better traction.

Reply:
We also run sphinx with `-W` in our options https://github.com/matplotlib/matplotlib/blob/512aecb3ba330a46c09f42683ae1e90d8302dcff/doc/Makefile#L5
so it is odd that astropy is failing but we are not...

I suspect difference is sphinx versions (we are currently pinning to sphinx < 2 due to weird spacing at TOC and some breakage (which may be this)).

It is also odd that this is not linking an the 'previous' link in the top right is the thing we want this to have linked to.


Agree with @QuLogic , this should get it's own issue.